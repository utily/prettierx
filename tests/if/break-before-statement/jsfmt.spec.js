run_spec(__dirname, ["babel"], { breakBeforeStatement: "conditionals" });
run_spec(__dirname, ["babel"], { breakBeforeStatement: "always" });
