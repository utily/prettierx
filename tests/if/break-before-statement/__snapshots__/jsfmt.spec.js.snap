// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`if.js - {"breakBeforeStatement":"always"} format 1`] = `
====================================options=====================================
breakBeforeStatement: "always"
parsers: ["babel"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
for (let i = 0; i < 2; i++) if (true) break

if (true) { console.log("test") }

for (const key in { p: 1 })
    if (object.hasOwnProperty(key))
        console.log(object[key]);

=====================================output=====================================
for (let i = 0; i < 2; i++)
  if (true)
    break;

if (true) {
  console.log("test");
}

for (const key in { p: 1 })
  if (object.hasOwnProperty(key))
    console.log(object[key]);

================================================================================
`;

exports[`if.js - {"breakBeforeStatement":"conditionals"} format 1`] = `
====================================options=====================================
breakBeforeStatement: "conditionals"
parsers: ["babel"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
for (let i = 0; i < 2; i++) if (true) break

if (true) { console.log("test") }

for (const key in { p: 1 })
    if (object.hasOwnProperty(key))
        console.log(object[key]);

=====================================output=====================================
for (let i = 0; i < 2; i++)
  if (true)
    break;

if (true) {
  console.log("test");
}

for (const key in { p: 1 })
  if (object.hasOwnProperty(key))
    console.log(object[key]);

================================================================================
`;
